### PR TITLE
Update getting_started_deploy.md

### PR DESCRIPTION
**Pull request by theodor2311**
Adding --dry-run=client to suppress the deprecated warning.

### CODE REVIEW

**Comment by theodor2311** [comment]
I signed it

File: prow/getting_started_deploy.md [hunk 1]
```diff
@@ -354,14 +354,14 @@ Now run the following to update the configmap.
 
 ```sh
 $ kubectl create configmap config \
-  --from-file=config.yaml=path/to/config.yaml --dry-run -o yaml | kubectl replace configmap config -f -
+  --from-file=config.yaml=path/to/config.yaml --dry-run=client -o yaml | kubectl replace configmap config -f -
```
Comment (alvaroaleman) [inline prow/getting_started_deploy.md:357]: why not `--dry-run=server`, thats the better check?

Comment (theodor2311) [reply prow/getting_started_deploy.md]: Thank you for the suggestion, changed to --dry-run=server instead.

**Review by alvaroaleman — APPROVED** [review]
/label tide/merge-method-squash

**Comment by alvaroaleman** [comment]
Thanks!

**Pull request merged**